demosplan/DemosPlanAssessmentTableBundle/Logic/AbstractRpcStatementBulkAction.php

```diff
@@ -99,6 +100,7 @@ abstract class AbstractRpcStatementBulkAction implements RpcMethodSolverInterfac
      * @var TransactionService
      */
     private $transactionService;
+    protected StatementDeleter $statementDeleter;
```

Comment:
Using `protected readonly StatementDeleter $statementDeleter` in the constructor would be more concise.

Reply:
a6b381f0b78add7ef5b84ba014ad19745b646ac0

Reply:
What I meant was the usage of [constructor property promotion](https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion) in conjunction with [readonly properties](https://www.php.net/manual/en/language.oop5.properties.php#language.oop5.properties.readonly-properties).